backend/cds/migrations/0001_initial.py

```diff
@@ -22,7 +22,7 @@ class Migration(migrations.Migration):
                         verbose_name="ID",
                     ),
                 ),
-                ("lecture_id", models.IntegerField()),
+                ("lecture_id", models.IntegerField(unique=True)),
```

Comment:
why you changed that? 🤔

Reply:
I accidentally removed the wrong migration earlier and undone the initial one. Did not notice that it put extra stuff there that is not needed. I am going to remove this. Thanks for the notice.